Python/symtable.c

```diff
@@ -70,6 +70,9 @@
 #define DUPLICATE_TYPE_PARAM \
 "duplicate type parameter '%U'"
 
+#define DUPLICATE_ARGUMENT \
+"duplicate argument '%U' in function definition"
+
```

Comment:
Please move this back:

```suggestion
```

Reply:
Oh yes, it was something that I started working on but I forgot to cancel that move. Thanks!